### ISSUE TITLE
Configuration value from new options structure is overridden by deprecated options

### ISSUE THREAD

**Issue by kwn** [getsentry/sentry-symfony#70]
I'm trying to use a new options structure to set up the environment name (`sentry.options.environment`) and it seems it doesn't work properly. The structure I've got looks like that:

```yml
sentry:
  dsn: http://xxxxxx
  options:
    environment: staging
```

Unfortunately all errors I receive in sentry are tagged with the environment name of "prod" which seems to be the default value for the symfony application (running through `web/app.php` front controller). I was wondering if sentry bundle still defaults to the deprecated option (`sentry.environment`). I think I found the issue here:

https://github.com/getsentry/sentry-symfony/blob/ea4cf2356a286319a3fc9a6326bdd852ae75d4ad/src/Sentry/SentryBundle/Resources/config/services.yml#L7

It seems the client object is constructed with options from the `sentry.options` and then some of options are overridden by deprecated values (for example `sentry.environment`);

**Comment by Jean85** [comment]
I've just tested it locally with
```
sentry:
  dsn: "******"
  options:
    environment: "%kernel.environment%"
```
and it works. Thanks anyway because I've found a different bug, which triggers an error when both are used at the same time but with identical values, and I will fix it ASAP.

But still, it works for me.

Are you sure that you are not watching errors from prod? You have to set debug at false to avoid Symfony catching all the errors and preventing Sentry from sending events.

**Comment by Jean85** [comment]
This was fixed in releases 0.8.4 and .6
